File: app/src/main/java/com/sabo/sabostorev2/ui/home/HomeViewModel.java
package com.sabo.sabostorev2.ui.home;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

import com.sabo.sabostorev2.Model.ItemStoreModel;

import java.util.List;

public class HomeViewModel extends ViewModel {

    private MutableLiveData<List<ItemStoreModel>> mutableLiveData;

    public HomeViewModel() {
        mutableLiveData = new MutableLiveData<>();
    }

    public MutableLiveData<List<ItemStoreModel>> getMutableLiveData() {
        return mutableLiveData;
    }

    public void setMutableLiveData(List<ItemStoreModel> itemStoreModelList) {
        mutableLiveData.setValue(itemStoreModelList);
    }
}

File: app/src/main/java/com/sabo/sabostorev2/ui/Home/HomeViewModel.kt
package com.sabo.sabostorev2.ui.Home

import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel
import com.sabo.sabostorev2.Model.Item.ItemStoreModel
import com.sabo.sabostorev2.Model.Item.ItemsModel

class HomeViewModel : ViewModel() {
    val mutableLiveHomeCategories: MutableLiveData<List<ItemStoreModel>> = MutableLiveData()
    val mutableLivePopular : MutableLiveData<List<ItemsModel>> = MutableLiveData()

    fun setMutableLiveHomeCategories(itemStoreModelList: List<ItemStoreModel>) {
        mutableLiveHomeCategories.value = itemStoreModelList
    }

    fun setMutableLivePopular(itemsModelList: List<ItemsModel>){
        mutableLivePopular.value = itemsModelList
    }
}